libraries/AP_RangeFinder/AP_RangeFinder_Benewake.cpp

```diff
@@ -123,16 +122,16 @@ bool AP_RangeFinder_Benewake::get_reading(uint16_t &reading_cm, bool &signal_ok)
                         } else {
                             // TF02 provides signal reliability (good = 7 or 8)
                             dist_reliable = (linebuf[6] >= 7);
-                        }
-                        if (dist_reliable) {
-                            sum_cm += dist;
-                            count++;
+
+                            if (dist_reliable) {
```

Comment:
This is wrong, TFmini doesn't add distance to sum_cm.

Reply:
My bad, logic corrected.